src/sc_kpm/utils/common_utils.py

```diff
@@ -133,7 +134,7 @@ def get_element_by_role_relation(src: ScAddr, rrel_node: ScAddr) -> ScAddr:
     return search_result.get(ScAlias.ELEMENT.value) if search_result else ScAddr(0)
 
 
-def get_link_content(link: ScAddr) -> str:
+def get_link_content(link: ScAddr) -> Union[str, int]:
     content_part = client.get_link_content(link)
     return content_part[0].data
```

Comment:
Is _client.get_link_content(link)_  return string and integer types?

Reply:
Yes, absolutely correct. But it doesn't work with float and binary